### PR TITLE
Bug fix/render correct battle

### PR DESCRIPTION
**Pull request by boscohyun**
### Description

1. Check rendered information of `HackAndSlash`, `MimisbrunnrBattle` and `RankingBattle` for render correct battle results.
2. Fix minor bug of `RankingBoard` UI

### Related Links

1. [[전투 랜더] HAS, 미미르의샘 로딩 중에 정확히 기다리고 있는 전투의 랜더만 동작시키기](https://app.asana.com/0/1141562434100787/1200814851909847/f)
2. [HAS Repeat 사용할 때 다른 아바타의 전투 스테이지 기록이 그대로 사용되어 전투하는 경우](https://app.asana.com/0/1141562434100787/1200867409730006/f)

### CODE REVIEW

File: nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs [hunk 1]
```diff
@@ -663,6 +663,13 @@ private void ResponseHackAndSlash(ActionBase.ActionEvaluation<HackAndSlash> eval
         {
             if (eval.Exception is null)
             {
+                if (eval.Action.Id != Game.Game.instance.ActionManager.LastBattleActionId ||
+                    eval.Action.worldId != Game.Game.instance.ActionManager.LastBattleWorldId ||
+                    eval.Action.stageId != Game.Game.instance.ActionManager.LastBattleStageId)
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs:668]:
```cs
if (eval.Action.Id != Game.Game.instance.ActionManager.LastBattleActionId ||
    eval.Action.worldId != Game.Game.instance.ActionManager.LastBattleWorldId ||
    eval.Action.stageId != Game.Game.instance.ActionManager.LastBattleStageId)
```
This statement overlaps with if statement of `ResponseMimisbrunnr` method.
Why don't you make a method to check this?

Comment (boscohyun) [reply nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs:668]: OK I'll try.

**Comment by Namyujeong** [comment]
@boscohyun Is it possible to test in editor?

**Review by Unengine — APPROVED** [review]

**Review by Namyujeong — APPROVED** [review]
✅ Battle
✅ Mimisbrunnr
✅ Rankingbattle

**Pull request merged**